JRC Publications Repository.js

```diff
@@ -67,10 +67,21 @@ function getSearchResults(doc, checkOnly) {
 function detectWeb(doc, url) {
 	var type = ZU.xpath(doc, '//meta[@name="DC.type"]/@content');
 	if (type.length>0) {
-		if (mappingTable[type[0].value]) {
-			return mappingTable[type[0].value];
-		} else {//generic fallback
-			Z.debug('Unrecognized item type: ' + type[0].value);
+		if (mappingTable[type[0].textContent]) {
+			return mappingTable[type[0].textContent];
```

Comment:
We should use `.value` for attributes https://developer.mozilla.org/en-US/docs/Web/API/Attr